Dockerfile

```diff
@@ -18,10 +18,15 @@ FROM node:lts-alpine as service
 
 WORKDIR /dscp-matchmaker-api
 
+RUN apk add --update coreutils
```

Comment:
without this `dscp-process-management` fails because it uses `#!/usr/bin/env -S node --no-warnings`